env.ts

```
@@ -10,9 +10,9 @@ const envSchema = z.object({
 		message: 'MAIN_URL must be a valid URL (e.g., http://localhost:3000)'
 	}).optional().default('http://localhost:3000'),
 
-	// Database
-	DATABASE_URL: z.string().url({
-		message: 'DATABASE_URL must be a valid PostgreSQL connection string'
+	// Backend
+	API_URL: z.string().url({
+		message: 'API_URL must be a valid URL'
 	}),
```

Biome (1.9.4)

[error] This property value named API_URL is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named API_URL


(lint/suspicious/noDuplicateObjectKeys)